docker/images/runners/n8n-task-runners.json

```diff
@@ -0,0 +1,53 @@
+{
+	"task-runners": [
+		{
+			"runner-type": "javascript",
+			"workdir": "/home/runner",
+			"command": "/usr/bin/node",
+			"args": [
+				"--disallow-code-generation-from-strings",
+				"--disable-proto=delete",
+				"/opt/runners/task-runner-javascript/dist/start.js"
+			],
+			"health-check-server-port": "5681",
+			"allowed-env": [
+				"PATH",
+				"GENERIC_TIMEZONE",
+				"N8N_RUNNERS_GRANT_TOKEN",
```

Comment:
[Here](https://github.com/n8n-io/task-runner-launcher/blob/main/docs/setup.md#environment-variables) at the bottom mentions the env vars that cannot be disallowed because they're operationally needed, so no need to explicitly allow them.